packages/compliance-tests/package.json

```diff
@@ -45,6 +45,8 @@
     "it-pipe": "^1.1.0",
     "libp2p-interfaces": "^1.1.0",
     "multiaddr": "^10.0.0",
+    "multihashes": "^4.0.3",
+    "cids": "^1.1.9",
```

Comment:
`cids` is deprecated, please use `multiformats` instead.

The CID implementation can be imported from `multiformats/cid`